runtime/autoload/help.vim

```diff
@@ -0,0 +1,38 @@
+function! s:synnames(...) abort
+  if a:0
+    let [line, col] = [a:1, a:2]
+  else
+    let [line, col] = [line('.'), col('.')]
+  endif
+  return reverse(map(synstack(line, col), 'synIDattr(v:val,"name")'))
+endfunction
+
+function! HelpTopic()
+  let col = col('.') - 1
+  while col && getline('.')[col] =~# '\k'
+    let col -= 1
+  endwhile
+  let pre = col == 0 ? '' : getline('.')[0 : col]
+  let col = col('.') - 1
+  while col && getline('.')[col] =~# '\k'
+    let col += 1
+  endwhile
+  let post = getline('.')[col : -1]
+  let syn = get(synnames(), 0, '')
+  let cword = expand('<cword>')
+  if syn ==# 'vimFuncName'
```

Comment:
The entire function is for `.vim` files: this and the following conditions will never match in a help file. I think this needs to be altered: topic for functions should be opened if current word is immediately followed by a parenthesis.